backend/src/main/java/com/alex/eshop/entity/Item.java

```diff
@@ -10,9 +10,10 @@ public class Item {
     private String name;
     @ManyToOne
     private Category category;
-
     private String description;
     private String imageSrc;
+    @Transient
+    private Boolean hasImage;
```

Comment:
`Boolean hasImage;` can be removed, as you can replace it with `imageSrc == null || imageSrc.isEmpty()` or using util method `ObjectUtils.isEmpty()` or `StringUtils.isBlank()`